app/src/main/res/layout/fragment_nav_ads.xml

```diff
@@ -115,6 +115,18 @@
             android:text="@string/action_show"
             android:visibility="gone" />
 
+        <TextView
```

Comment:
For this error message I'd add a small label like the other components have that states this is the error message. Similar to HyBid demo app. Also I'd add the same margins the other components have.

Reply:
Added label

Reply:
Done